spec/system/jobseekers_can_register_spec.rb

```diff
@@ -15,6 +15,7 @@
 
     click_on I18n.t("jobseekers.registrations.check_your_email.resend_link"), visible: false
     visit first_link_from_last_mail
+    click_on "Confirm"
```

Comment:
Would you `confirm_email_address` here too?

Reply:
While they do the same thing, I wanted to be more explicit about the two steps here as this is what's actually under test, whereas in the other cases it's a step on the way to testing something else.